docs/vz235_mellow/top_cover/top_cover.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
+parent: 8. Top cover and doors panels
+grand_parent: Vz235 - Mellow Kit
+has_toc: false
+nav_order: 1
+has_children: false
+permalink: /vz235_mellow/top_cover/latches
+---
+
+# 8.1 Top Cover Assembly
+
+## Overview
+
+![Overview](../../assets/images/manual/vz235_printed/top_cover/top_cover_overview.png)
+
+## BOM
+
+| Material                    | Quantity | Notes                                                |
+| --------------------------- | -------- | ---------------------------------------------------- |
+| M3 x 8 BHCS                 | 8        |                                                      |
+| M3 Nut                      | 8        |                                                      |
+| M4 x 10 SHCS                | 8        |                                                      |
+| M4 hammer t-nut             | 8        |                                                      |
+| 2 x 8mm Magnet              | 8        |                                                      |
+| 395\*405\*3mm acrylic panel | 1        | Refer to the .step file for placement of the cutouts |
+
+## STL's
+
+| File name                      | Amount to print |
+| ------------------------------ | --------------- |
+| [Top cover holder]             | 4               |
+| [Top cover middle part]        | 4               |
+| [Top cover top part]           | 4               |
+
+### Step 1
```

Comment:
The headings look rather small with the h3 heading size. Would h2 solve this, or could a styling change be necessary to make the steps clearer to the user? 

Been considering making headings "looser" and have more margin above them if they are not the first one in the page.

Reply:
I used the same size that was already there.
Not sure what you mean about making them "looser". The spacing of them is as per the markdown standard

Reply:
This comment was mostly conversation oriented. My bad for not communicating that clearly.